util/reggen/reg_top.sv.tpl

```diff
@@ -243,31 +243,25 @@ module ${mod_name} (
 
   // Create steering logic
   always_comb begin
-    reg_steer = ${num_dsp-1};       // Default set to register
+    // Default set to register
+    reg_steer = ${num_dsp-1};
```

Comment:
Couldn't this go into `default:` for the case statement below?

Reply:
Indeed, let's do that!